runtime/onert/core/include/exec/Executors.h

```diff
@@ -36,13 +40,11 @@ class Executors
   Executors(const Executors &) = delete;
   Executors(Executors &&) = default;
 
-  // TODO Use Executor index
-  void emplace(ir::SubgraphIndex idx, std::unique_ptr<IExecutor> exec)
-  {
-    _executors.emplace(idx, std::move(exec));
-  }
+  void emplace(const ir::SubgraphIndex &idx, std::unique_ptr<IExecutor> exec);
+
+  IExecutor *at(const ExecutorIndex &idx) const { return _executors.at(idx).get(); }
```

Comment:
@hseok-oh I thought more. Is it possible not to expose `ExecutorIndex` at all? 

**Interface**

`(ModelIndex, SubgraphIndex)`  -> `IExecutor *`
```suggestion
  IExecutor *at(std::pair<ir::ModelIndex, ir::SubgraphIndex>) const;
```

Reply:
Or you may use two parameters instead of `std::pair`.